Formula/libgr.rb

```diff
@@ -0,0 +1,48 @@
+class Libgr < Formula
+  desc "GR framework: a graphics library for visualisation applications"
+  homepage "https://gr-framework.org/"
+  url "https://github.com/sciapp/gr/archive/v0.37.0.tar.gz"
+  sha256 "9120c4215cd2c10239f45f4cbf79dcb199a9e7e3bbbf36e5ce8f4e5eb2e70e24"
+  depends_on :xcode => :build
```

Comment:
Why does it need Xcode to build?

Reply:
One important piece is the `GKSTerm.app` which is what is going to render the graphs in a native window. That's the default behavior. It can alternatively run headless and just generate files on disk like `.png`, `.jpeg` or `.mov` for animations.

Reply:
building `app`s is generally reserved for casks.

Reply:
I'm not really sure where this fit, as the app itself can't be used directly, but only via the internal API.

This would have been a nice fit for the defunct `homebrew-science` repo.

Reply:
@fxcoudert @SMillerDev please let me know if you want me to convert this into a cask. I don't really think Cask is the best alternative here, but I'm happy with whatever you all decide

Reply:
I think it makes sense, then. @SMillerDev?